gstools/field/base.py

```diff
@@ -297,9 +296,8 @@ def plot(
                 "Specify 'scalar' or 'vector' before plotting."
             )
 
-        elif self.value_type == "scalar":
+        if self.value_type == "scalar":
```

Comment:
Is some super deep Python knowledge required to understand why elif is wrong here? Or is this a convention to separate error raising from other branching logic?

Reply:
See here: https://vald-phoenix.github.io/pylint-errors/plerr/errors/refactoring/R1720

Reply:
The rule simply says: no need for an `else` after a `raise`.